src/log.cpp

```diff
@@ -274,8 +275,39 @@ void Logger::log(LogLevel lev, const std::string &text)
 	const std::string label = getLevelLabel(lev);
 	const std::string timestamp = getTimestamp();
 	std::ostringstream os(std::ios_base::binary);
+
+	static const std::string use_logcolor = g_settings->get("log_color");
+
+	bool colored = use_logcolor == "tty" ? isatty(STDOUT_FILENO) : use_logcolor == "yes";
+	if (colored)
+		switch (lev) {
+			case 1:
```

Comment:
LL_* is the least significant problem here. `isatty` can be found as `_isatty` in `io.h` for Windows.
The console colors instead require a call of `SetConsoleTextAttribute` with the specific color parameters.